frontend/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <link href="../css/weather-icons.min.css" rel="stylesheet">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/date-fns/1.28.5/date_fns.min.js"></script>
+  <script src="./marcel.js"></script>
+</head>
+
+<body>
+  <p id="season"></p>
+  <i id="icon" class="wi wi-night-sleet"></i>
+  <p id="content"></p>
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+      overflow: hidden;
+      width: 100vw;
+      height: 100vh;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      flex-direction: column;
+    }
+
+    i {
+      font-size: 5em;
+      font-size: 25vw;
+    }
+
+    p {
+      font-size: 5em;
+      font-size: 10vw;
+      margin: .3em;
+    }
+
+    #season {
+      margin-bottom: 2em;
+    }
+  </style>
+
+  <script>
+    const { isToday, closestTo, isEqual } = dateFns
+    class Text extends Marcel.Plugin {
+      constructor() {
+        super({
+          defaultProps: {
+            text: '',
+            textAlign: 'center',
+            stylesvar: {}
+          },
+        })
+
+        this.temperature = document.getElementById('content')
+        this.icon = document.getElementById('icon')
+        this.season = document.getElementById('season')
+      }
+
+      mapIcon = key =>
+        ({
+          8: 'wi-day-sunny',
+          2: 'wi-day-thunderstorm',
+          3: 'wi-day-rain',
+          5: 'wi-day-rain',
+          6: 'wi-day-snow',
+          7: 'wi-day-cloudy',
+          9: 'wi-day-cloudy'
+        }[key] || 'wi-day-sunny')
```

Comment:
Perhaps put this map outside ? It's a constant, not sure it have to be recreated at each function call ?